web/db/dbConnect.ts

```diff
@@ -3,6 +3,8 @@ declare var global: any;
 import mongoose from "mongoose";
 import { consts } from "../utils/consts";
 
+mongoose.set("strictQuery", false);
```

Comment:
strictQuery options would be best set in each schema as appropriate, instead of globally.